javascript/Game.js

```diff
@@ -22,6 +24,26 @@ class Game {
   declareWinner() {
   }
 
-}
+  createCategories() {
+    let categoryKeys = Object.entries(data.categories);
+      let randoms = [];
+        for (var i = 0; i < categoryKeys.length; i++) {
```

Comment:
Any reason for using a for loop over a `forEach` here? Generally, developers will choose a `forEach` over a for loop because it's more readable and gives you easier access to the current element you're iterating over. Let me know if you ran into any issues with `forEach` - otherwise I'd consider switching over to that :)